app/javascript/modules/listingDetails/ListingDetailsPricingTable.tsx

```diff
@@ -200,6 +193,60 @@ const buildAccordions = (
   )
 }
 
+const buildHabitatText = (
+  groupedUnitsByOccupancy: GroupedUnitsByOccupancy[],
+  amiCharts: RailsAmiChart[]
+) => {
+  const habitatStringArray = []
+  const minAmiChartsValues = amiCharts.find((chart) => {
+    return chart.derivedFrom === "MinAmi"
+  })?.values
+
+  const maxAmiChartsValues = amiCharts.find((chart) => {
+    return chart.derivedFrom === "MaxAmi"
+  })?.values
+
+  /*
+   * GroupedUnitsByOccupancy() is already sorted
+   */
+  const minOccupancy = groupedUnitsByOccupancy[0]?.occupancy
+  const maxOccupancy = minOccupancy + 9
+
+  /*
+   * We want to display 9 rows for Habitat listings
+   */
+  for (let i = minOccupancy; i < maxOccupancy; i++) {
+    const minOccupancyChart = minAmiChartsValues.find((chart) => {
+      return chart.numOfHousehold === i
+    })
+
+    const maxOccupancyChart = maxAmiChartsValues.find((chart) => {
+      return chart.numOfHousehold === i
+    })
+
+    if (minOccupancyChart && maxOccupancyChart) {
+      habitatStringArray.push(
+        `${i} people household: $${minOccupancyChart?.amount?.toLocaleString()} to $${maxOccupancyChart?.amount?.toLocaleString()}`
```

Comment:
Will `i` ever be 1 or 0? If that is possible, then we will need to have a singular and plural string variant. E.g. "1 person household" vs "2 People Household"

Also, will this need to be run through translation?